pygmt/src/grdcontour.py

```diff
@@ -53,16 +51,17 @@ def grdcontour(self, grid, **kwargs):
           angle (col 3).
         - A fixed contour interval *cont_int* or a single contour with
           +\ *cont_int*.
-    annotation : str, int, or list
+    annotation : int or str
         Specify or disable annotated contour levels, modifies annotated
         contours specified in ``interval``.
 
         - Specify a fixed annotation interval *annot_int* or a
-          single annotation level +\ *annot_int*.
+          single annotation level **+**\ *annot_int*.
         - Disable all annotation with  **-**.
```

Comment:
Some descriptions (e.g., disable annotations with `-`) are outdated. Need to update the docstrings following the upstream documentation https://docs.generic-mapping-tools.org/6.5/grdcontour.html#a.